### PR TITLE
docs: v11 color token updates for loading component

### PR DESCRIPTION
**Pull request by johnbister**
This PR updates the color tokens on the style tab for the loading component.

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]

File: src/pages/components/loading/style.mdx [hunk 1]
```diff
@@ -8,11 +8,11 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
-| Class                  | Property         | Color token         |
-| ---------------------- | ---------------- | ------------------- |
-| `.bx--loading__svg`    | stroke           | `$interactive-04`   |
-| `.bx--loading--small`  | stroke           | `$interactive-04`   |
-| `.bx--loading-overlay` | background-color | `rgba($ui-02, 0.6)` |
+| Class                  | Property         | Color token    |
+| ---------------------- | ---------------- | -------------- |
+| `.bx--loading__svg`    | stroke           | `$interactive` |
+| `.bx--loading--small`  | stroke           | `$interactive` |
+| `.bx--loading-overlay` | background-color | `$overlay'     |
```
Comment (aagonzales) [inline src/pages/components/loading/style.mdx:15]: You've got the wrong `'` on the send of overlay so the visual style isn't showing up correctly. 
image

**Review by aagonzales — CHANGES_REQUESTED** [review]

File: src/pages/components/loading/style.mdx (hunk 1 above)
Comment (tw15egan) [reply src/pages/components/loading/style.mdx:15]:
```suggestion
| `.bx--loading-overlay` | background-color | `$overlay`     |
```

**Review by emyarod — APPROVED** [review]
looks good to me

**Review by aagonzales — APPROVED** [review]
Looks good!

**Pull request merged**